.github/workflows/integration-tests-pr.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       - name: Apply Calico Rules to LKE
         if: always()
         run: |
-          cd scripts && ./lke_calico_rules_e2e.sh
+          cd e2e_scripts/cloud_security_scripts/lke_calico_rules/ && ./lke_calico_rules_e2e.sh
```

Comment:
I think we need to recursively clone submodules during the checkout step in this file 👍 

See: https://github.com/linode/ansible_linode/actions/runs/9664128640/job/26658035956#step:12:70

Reply:
Nice catch! Thanks